src/test/java/io/confluent/connect/elasticsearch/ElasticsearchSinkConnectorConfigTest.java

```diff
@@ -0,0 +1,49 @@
+package io.confluent.connect.elasticsearch;
+
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class ElasticsearchSinkConnectorConfigTest {
+
+  private Map<String, String> props;
+
+  @Before
+  public void createProps() {
```

Comment:
Minor nit: maybe name this method `setup()`?

Reply:
Thanks guys. This is done.